### ISSUE TITLE
Use upstream sources

### ISSUE THREAD

**Issue by dper** [dper/aedict#1]
Where the upstream sources are not referenced directly, they should be. The author did a lot of repackaging, and building the app has historically involved downloading lots of packages from his server. This is bad because we don't know if his server is stable, what he did with those apps, and whether he has properly open-sourced them.

Where possible, we should use the upstream sources, such as the maven website, when installing dependencies.  Strings like `vyzivus0` and `http://baka.sk/maven2` are good ones to look for.  Most of those should be replaced.

In the long run, we can't be dependent on the author's private packages.  If we can't find a better source for a dependency, it would be good to find a different program to depend on.  This is easier said than done, though.

**Comment by mvysny** [comment]
Yup, some of the stuff had to be repackaged (lucene comes to mind), simply because Android build system crashed when it found java.awt-related classes in Lucene. However, in Aedict 3 I'm using vanilla lucene-core-3.6.2 which seem to work nicely.
